modules/ingester/downscale.go

```diff
@@ -0,0 +1,105 @@
+package ingester
+
+import (
+	"net/http"
+
+	kitlog "github.com/go-kit/log"
```

Comment:
Could we use the standard slog instead?

Reply:
I don't a strong opinion on loggers, but `slog` is not widely used in the codebase. IMO for consistency the go-kit logger makes more sense.